packages/babel-plugin/__tests__/stylex-perf-create-theme-test.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+'use strict';
+
+jest.autoMockOff();
+
+const { transformFileSync } = require('@babel/core');
+const stylexPlugin = require('../src/index');
+const path = require('path');
+
+const classNamePrefix = 'x';
+const defaultOpts = {
+  stylexSheetName: '<>',
+  unstable_moduleResolution: { type: 'haste' },
+  classNamePrefix,
+};
+
+function transform(file, opts = defaultOpts) {
+  const result = transformFileSync(file, {
+    filename: opts.filename || '/stylex/packages/TestTheme.stylex.js',
```

Comment:
Having a fake default filename here is a bit confusing, esp since we're passing in a "theme" in the test but the fallback is "Theme.stylex.js" which is a file suffix typically used for files containing defineVars exports